pkgs/games/klavaro/default.nix

```diff
@@ -1,17 +1,38 @@
-{ stdenv, fetchurl, makeWrapper, pkgconfig, intltool, curl, gtk3 }:
+{ stdenv, fetchurl, makeWrapper, pkgconfig, intltool, curl, gtk3, espeak }:
 
 stdenv.mkDerivation rec {
   name = "klavaro-${version}";
-  version = "3.08";
+  version = "3.09";
 
   src = fetchurl {
     url = "mirror://sourceforge/klavaro/${name}.tar.bz2";
-    sha256 = "0qmvr6d8wshwp0xvk5wbig4vlzxzcxrakhyhd32v8v3s18nhqsrc";
+    sha256 = "12gml7h45b1w9s318h0d5wxw92h7pgajn2kh57j0ak9saq0yb0wr";
   };
 
   nativeBuildInputs = [ intltool makeWrapper pkgconfig ];
   buildInputs = [ curl gtk3 ];
 
+  patches = [ (builtins.toFile "format-string" ''
```

Comment:
don't you think this is a rather, interesting way to patch things?
Is this your patch @dtzWill?

Reply:
Yes, this is mine :).  I like having small patch inline but can of course move to a separate file if preferred?